modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,269 @@
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Performance Addon Operator
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/openshift-kni/performance-addon-operator-index:4.7-snapshot
+  publisher: Red Hat
+  sourceType: grpc
+----
+
+. Create the `operator_operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+----
+
+. Create the `operator_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: "4.7"
+  name: performance-addon-operator
+  source: performance-addon-operator
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# TODO handle namespace with var
+
+resources:
+  - operator-namespace.yaml
+  - operator_operatorgroup.yaml
+  - operator_catalogsource.yaml
+  - operator_subscription.yaml
+----
+
+== Installing the Precision Time Protocol (PTP) operator
+
+Install the PTP Operator using the OpenShift Container Platform CLI or the web console.
+
+.Procedure
+
+. Create the `01_namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-ptp
+  labels:
+    openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
+----
+
+. Create the `02_operator_group.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: ptp-operators
+  namespace: openshift-ptp
+spec:
+  targetNamespaces:
+    - openshift-ptp
+----
+
+. Create the `03_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "4.6"
+  name: ptp-operator
+  source: "redhat-operators"
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - 01_namespace.yaml
+  - 02_operator_group.yaml
+  - 03_subscription.yaml
+----
+
+== Applying the Stream Control Transmission Protocol (SCTP) patch
+
+Load and enable the blacklisted SCTP kernel module on worker nodes in your cluster.
+
+.Procedure
+
+. Create the `is_ready.sh` executable file:
++
+[source,yaml]
+----
+#!/bin/sh
+
+# no output means that the new machine config wasn't picked by MCO yet
+if [ -z "$(oc get mcp worker-cnf -o jsonpath='{.spec.configuration.source[?(@.name=="load-sctp-module")].name}')" ]; then
+    exit 1
+fi
+
+oc wait mcp/worker-cnf --for condition=updated --timeout 1s
```

Comment:
Is `oc wait` a separate command? If so, it should be separated out into its own code block and have a preceding $ or # per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
From Vitaly - "oc wait is a separate command, but both blocks are a part of the same script. The first one is checks whether the second one can be used to check the configuration outcome or it’s to early. I would not separate them."